x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlExecutionInfo.java

```diff
@@ -247,7 +247,7 @@ public Iterator<? extends ToXContent> toXContentChunked(ToXContent.Params params
             ChunkedToXContentHelper.field(PARTIAL_FIELD.getPreferredName(), getClusterStateCount(Cluster.Status.PARTIAL)),
             ChunkedToXContentHelper.field(FAILED_FIELD.getPreferredName(), getClusterStateCount(Cluster.Status.FAILED)),
             // each Cluster object defines its own field object name
-            ChunkedToXContentHelper.xContentFragmentValuesMapCreateOwnName("details", clusterInfo),
+            ChunkedToXContentHelper.object("details", clusterInfo.values().iterator()),
```

Comment:
Is this equivalent? The implementations are surely different, but maybe the end result is the same. Just checking.

Reply:
It is equivalent - the chunk counts dont change, and the tests still all pass (notice `xContentFragmentValuesMapCreateOwnName` is just calling `getValue.toXContent`, ignoring the key entirely)